### ISSUE TITLE
Figure 1 example 1 implies it passes adjacent contrast but that aspect might not apply to this SC 2.4.11

### ISSUE THREAD

**Issue by mraccess77** [w3c/wcag#2087]
The understanding doc states "Three example indicators, an outline separated from the button, an inner outline, and a thick outline.
Figure 1 Examples of outlines that would pass easily on a light background."
![image](https://user-images.githubusercontent.com/5640755/137416518-f324047e-db96-485f-94d8-f2d260c3f7be.png)

However, because it's an outset indicator - the adjacent contrast requirements would not seem to apply in this case- SC 1.4.11 would cover the color adjacent to the component.

**Comment by alastc** [comment]
Ok, now I'm more confused, that image is from the 2.4.11.

The adjacent bullet of 2.4.11 doesn't apply to that case, true, but that's the point, it passes! And because it is not adjacent to the component, that's why it can be a dark outline around a dark component and be 1px thick. 

It also avoids issue with 1.4.11, but that's good, we wouldn't want a good example to fail another SC. 

I think I still don't understand what the problem is.

**Comment by mraccess77** [comment]
Both SC 1.4.11 and 2.4.11 cover adjacent contrast.  I think where we landed is that:
* SC 1.4.11 would cover adjacent contrast in or out of the component with no exceptions for thickness.  
* SC 2.4.11's adjacency was aimed at outside but touching the component but also has an exception for 2px thickness.  
So a realistic question might be if the indicator is not touching the component and is not 2px wide then it must meet >= 3:1 contrast from SC 1.4.11 as the 2px provision of SC 2.4.11 doesn't apply?
Or does adjacent to the component mean it doesn't have to touch?  

Passing the 2px requirement in SC 2.4.11 doesn't really help if you already fail SC 1.4.11.

**Comment by alastc** [comment]
> - SC 1.4.11 would cover adjacent contrast in or out of the component with no exceptions for thickness.

Yes, with no requirement for size or thickness. That doesn't stop focus-appearance from adding requirements.

> - SC 2.4.11's adjacency was aimed at outside but touching the component but also has an exception for 2px thickness.

Yes. For 1.4.11, the adjacency is to the component OR the background, if you remember the example with the odd yellow box not aligned very well, it could contrast with either the component background or the page background.

> if the indicator is not touching the component and is not 2px wide then it must meet >= 3:1 contrast from SC 1.4.11 as the 2px provision of SC 2.4.11 doesn't apply?

Yes, in terms of adjacency, that scenario would be with the outer (page) background. However, that should be the same as the 'change of contrast' bullet. 

It is possible you could have content next to the control, so it is adjacent to the outside of the outline, but I don't think that's a problem.

> Passing the 2px requirement in SC 2.4.11 doesn't really help if you already fail SC 1.4.11.

We certainly shouldn't create a clash, but I don't see one.

> the wording seems to imply it also is applicable and pass the adjacency requirements of SC 2.4.11.

It does pass the requirement. It does so by not being adjacent, which is a perfectly good way of doing it.
